force-app/main/default/classes/CRLP_Batch_Base.cls

```diff
@@ -503,6 +506,9 @@ public abstract class CRLP_Batch_Base {
             get {
                 if (maxDaysAgo == null) {
                     maxDaysAgo = getMaxIntegerValueFromRollupsForType(CRLP_Operation.TimeBoundOperationType.Days_Back);
+                    if (maxDaysAgo == null && base.customizableRollupSettings.LimitRecalculatedRecurringDonations__c) {
```

Comment:
I think, like the issue I noted earlier, if the new field for `LimitRecalculatedRecurringDonations__c` is null ,this might cause an NPE.